helm/cert-manager/templates/podlogs.yaml

```diff
@@ -0,0 +1,91 @@
+{{- if and .Values.monitoring .Values.monitoring.podLogs .Values.monitoring.podLogs.enabled }}
+---
+# PodLogs for cert-manager core
+apiVersion: monitoring.grafana.com/v1alpha2
+kind: PodLogs
+metadata:
+  name: cert-manager-logs
+  namespace: kube-system
```

Comment:
since this runs in kube-system, did you check that the logs aren't already being collected?